scripts/fdbshow

```diff
@@ -0,0 +1,192 @@
+#!/usr/bin/python
+"""
+    Script to show MAC/FDB entries learnt in Hardware
+    
+    usage: fdbshow [-p PORT] [-v VLAN]
+    optional arguments:
+      -p,  --port              FDB learned on specific port: Ethernet0
+      -v,  --vlan              FDB learned on specific Vlan: 1000
+  
+    Example of the output:
+    admin@str~$ fdbshow
+      No.    Vlan  MacAddress         Port
+    -----  ------  -----------------  ----------
+        1    1000  7C:FE:90:80:9F:05  Ethernet20
+        2    1000  7C:FE:90:80:9F:10  Ethernet40
+        3    1000  7C:FE:90:80:9F:01  Ethernet4
+        4    1000  7C:FE:90:80:9F:02  Ethernet8
+    Total number of entries 4
+    admin@str:~$ fdbshow -p Ethernet4
+      No.    Vlan  MacAddress         Port
+    -----  ------  -----------------  ---------
+        1    1000  7C:FE:90:80:9F:01  Ethernet4
+    Total number of entries 1
+    admin@str:~$ fdbshow -v 1001
+    1001 is not in list
+
+"""
+
+import swsssdk
+import argparse
+import json
+import sys
+
+from tabulate import tabulate
+from natsort import natsorted
+
+class FdbShow(object):
+
+    HEADER = ['No.', 'Vlan', 'MacAddress', 'Port']
+    FDB_COUNT = 0
+
+    def __init__(self):
+        super(FdbShow,self).__init__()
+        self.db = swsssdk.SonicV2Connector(host="127.0.0.1")
```

Comment:
>host [](start = 43, length = 4)

We prefer using unix socket.

Reply:
As discussed, this format is used by other standalone scripts which also handles the permission issue.